Assets/Scripts/UI/DialogBox/Settings/DialogBoxSettings.cs

```diff
@@ -100,6 +107,10 @@ public void OnLangageToggle()
         langDropDown.SetActive(langToggle.isOn);
     }
 
+    public void OnDeveloperModeToggle()
+    {
+    }
```

Comment:
Is `OnDeveloperModeToggle()` supposed to do something? Otherwise, maybe it should be removed. If you remove it, take out the listener as well.